<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EEcamera</class>
 <widget class="QWidget" name="EEcamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="8,2">
   <item>
    <widget class="QLabel" name="imgShow">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="4,3,1,1,0,3">
     <item>
      <widget class="QGroupBox" name="cameraBox">
       <property name="title">
        <string>Camera</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="cameraOpenButton">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="camerCloseButton">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="endButton">
            <property name="text">
             <string>End</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="pathEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="captureOnce">
          <property name="text">
           <string>Capture Once</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="roiBox">
       <property name="title">
        <string>ROI</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="cx">
            <property name="text">
             <string>cx</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="cy">
            <property name="text">
             <string>cy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>w</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="h">
            <property name="text">
             <string>h</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLineEdit" name="cxEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="cyEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="wEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="hEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="roiSetButton">
            <property name="text">
             <string>SetROI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="roiDelButton">
            <property name="text">
             <string>DelROI</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QPushButton" name="calibraButton">
         <property name="text">
          <string>Calibrate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="keyholeClickButton">
         <property name="text">
          <string>Keyhole</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QPushButton" name="modelLoadButton">
         <property name="text">
          <string>Load Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mRoiButton">
         <property name="text">
          <string>roi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QPushButton" name="addClickButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="undoClickButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearClickButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>no Calibrate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Width:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gapWidthEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Dev:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="DeviationEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>axis:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="axisBox">
             <item>
              <property name="text">
               <string>x</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>y</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>PTrace:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="pTraceBox">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Center</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Side</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>pre_mask:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="preMaskBox">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Prob</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Binary</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
